tests/mobly_base_class_test.py

```diff
@@ -199,24 +199,6 @@ def test_something(self):
                             "Skipped 0, Unknown 0")
```

Comment:
We don't have any tests that tested "return False" from tests themselves? Only setup?